spring-data-examples/eviction/src/main/java/example/springdata/gemfire/eviction/EvictionClient.java

```diff
@@ -0,0 +1,36 @@
+package example.springdata.gemfire.eviction;
+
+import org.apache.geode.cache.Region;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.WebApplicationType;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class EvictionClient {
+
+    public static void main(String[] args) {
+        new SpringApplicationBuilder(EvictionClient.class).web(WebApplicationType.NONE).build().run(args);
+    }
+
+    @Bean
+    public ApplicationRunner runner(OrderRepository orderRepository, @Qualifier("Orders") Region<Long, Order> orderRegion) {
+        return args -> {
+            final int evictionThreshold = 10;
+            System.out.println("Inserting " + (evictionThreshold + 3) + " entries");
```

Comment:
Why system out? Is there no Logger?

Reply:
The logging that was in the examples wasn't working and I thought this would be simpler, but I can convert it all to logging if we want.

Reply:
Logging would be the better solution.

Reply:
Replaced System out with logging